environment/src/main/java/com/sequenceiq/environment/environment/v1/EnvironmentController.java

```diff
@@ -369,6 +369,14 @@ public FlowIdentifier postStopByCrn(@ValidCrn(resource = CrnResourceDescriptor.E
         return environmentStopService.stopByCrn(crn);
     }
 
+    @Override
+    @CheckPermissionByResourceName(action = AuthorizationResourceAction.ROTATE_SALTUSER_PASSWORD_ENVIRONMENT)
+    public void rotateSaltPasswordByName(@ResourceName String name) {
```

Comment:
so same here, authz based on crn would be better, if name is not strongly required by business reasons